pkg/vault/vault.go

```diff
@@ -59,9 +60,15 @@ func (c *Client) ReadFrom(path string) *ReadClient {
 	return r
 }
 
+var breaker = circuitbreaker.NewCircuitBreaker("vault-circuit-breaker")
```

Comment:
Here the name of the circuit breaker is vaul-circuit-breaker, but for OICD only oicd, there the naming pattern is not quite consistent.

Reply:
Or better remove the name, because there is only one proxy type running.